installer.sh

```diff
@@ -0,0 +1,442 @@
+: "${INSTALL_DIR:=}"
+
+# Platform.sh CLI version to install
+: "${VERSION:=}"
+
+# macOS specifics
+: "${BREW_TAP:=platformsh/tap}"
+: "${BREW_FORMULA:=platformsh/tap/platformsh-cli}"
+
+# GitHub token check
+: "${GITHUB_TOKEN:=}"
+
+# global variables
+binary="platform"
+cmd_shasum=""
+cmd_sudo=""
+dir_bin="/usr/bin"
+footer_notes=""
+has_sudo=""
+kernel=""
+machine=""
+version=""
+
+# create a log file where every output will be pipe to
+pipe=/tmp/platformsh-install-$$.tmp
+mkfifo $pipe
+tee < $pipe $INSTALL_LOG &
+exec 1>&-
+exec 1>$pipe 2>&1
+trap 'rm -f $pipe' EXIT
+
+function output {
+    style_start=""
+    style_end=""
+    if [ "${2:-}" != "" ]; then
+    case $2 in
+        "success")
+            style_start="\033[0;32m"
+            style_end="\033[0m"
+            ;;
+        "error")
+            style_start="\033[31;31m"
+            style_end="\033[0m"
+            ;;
+        "info"|"warning")
+            style_start="\033[33m"
+            style_end="\033[39m"
+            ;;
+        "heading")
+            style_start="\033[1;33m"
+            style_end="\033[22;39m"
+            ;;
+        "comment")
+            style_start="\033[2m"
+            style_end="\033[22;39m"
+            ;;
+    esac
+    fi
+
+    builtin echo -e "${style_start}${1}${style_end}"
+}
+
+function exit_with_error() {
+    output "+--------------------------------------------------+" "error"
+    output "|                                                  |" "error"
+    output "|              Installation failed                 |" "error"
+    output "|                                                  |" "error"
+    output "+--------------------------------------------------+" "error"
+
+    output "\nGet help with your Platform.sh CLI installation:" "heading"
+    output "  Inspect the logs: ${INSTALL_LOG}"
+    output "  Read the docs: https://docs.platform.sh/administration/cli.html"
+    output "  Get help: https://platform.sh/support"
+
+    exit 1
+}
+
+function intro() {
+    output "+--------------------------------------------------+" "heading"
+    output "|                                                  |" "heading"
+    output "|           Platform.sh CLI Installer              |" "heading"
+    output "|                                                  |" "heading"
+    output "+--------------------------------------------------+" "heading"
+
+    output "\nChecking environment" "heading"
+}
+
+function outro() {
+    output ""
+    output "+--------------------------------------------------+" "success"
+    output "|                                                  |" "success"
+    output "| Platform.sh CLI has been installed successfully. |" "success"
+    output "|                                                  |" "success"
+    output "+--------------------------------------------------+" "success"
+
+    output "\nWhat's next?" "heading"
+
+    output "  Get started with: platform welcome" "output"
```

Comment:
The `welcome` command is run from `platform`

```suggestion
    output "  Get started by running: platform" "output"
```

```suggestion
    output "  To use the CLI, run: platform" "output"
```